stable/insights-agent/README.md

```diff
@@ -52,6 +52,7 @@ Parameter | Description | Default
 `cronjobs.successfulJobHistoryLimit` | Number of successful jobs to keep in history for each report | 2
 `cronjobs.nodeSelector` | Node selector to use for cronjobs | null
 `cronjobs.runJobsImmediately` | Run each of the reports immediately upon install of the Insights Agent | true
+`cronjobs.dnsPolicy` | Add pod DNS policy. If dnsPolicy is not explicitly specified, then “ClusterFirst” is used |
```

Comment:
Let's get rid of `If dnsPolicy is not explicitly specified, then “ClusterFirst” is used` - I think this might be configured on a per-cluster basis

Reply:
sure let me remove it ... but I think `Default` value is the one that follows the `node` [configuration.](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy)